etc/config/swift.amazon.properties

```diff
@@ -1,12 +1,24 @@
-compilers=&swift
+compilers=&swift:&arm-swift
 demangler=/opt/compiler-explorer/swift-6.0.3/usr/bin/swift-demangle
 defaultCompiler=swift603
 objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 
+#################################
+# Compiler Groups
+#################################
+
+# swift (x86_64)
 group.swift.compilers=swift311:swift402:swift403:swift41:swift411:swift412:swift42:swift50:swift51:swift52:swift53:swift54:swift55:swift56:swift57:swift58:swift59:swift510:swift603:swiftdevsnapshot:swiftnightly
 group.swift.isSemVer=true
+group.swift.groupName=swift (x86-64)
 group.swift.baseName=x86-64 swiftc
 
+# arm-swift (aarch64)
+group.arm-swift.compilers=arm-swift603
+group.arm-swift.isSemVer=true
+group.arm-swift.groupName=swift (aarch64)
+group.arm-swift.baseName=aarch64 swiftc
```

Comment:
just to confirm my understanding of these fields:

- `groupName`: controls the 'section header' in the compiler dropdown list in the UI. it appears to get fully capitalized and the sections are sorted lexicographically.
- `baseName`: controls the prefix of the specific compiler string name that is merged with the semver value. so the newly-added compiler here would be expected to render as `aarch64 swiftc 6.0.3`

Reply:
That's right, yes :)